examples/loadbalancer/loadbalancer.go

```diff
@@ -0,0 +1,86 @@
+package main
+
+import (
+	"context"
+	"fmt"
+	"os"
+
+	"github.com/stackitcloud/stackit-sdk-go/core/config"
+	"github.com/stackitcloud/stackit-sdk-go/core/utils"
+	"github.com/stackitcloud/stackit-sdk-go/services/loadbalancer"
+)
+
+func main() {
+	// Specify the project ID
+	projectId := "PROJECT_ID"
+
+	// Create a new API client, that uses default authentication and configuration
+	loadbalancerClient, err := loadbalancer.NewAPIClient(
+		config.WithRegion("eu01"),
+	)
+	if err != nil {
+		fmt.Fprintf(os.Stderr, "Creating API client: %v\n", err)
+		os.Exit(1)
+	}
+
+	// List the load balancer instances for your project
+	listInstancesResp, err := loadbalancerClient.ListLoadBalancers(context.Background(), projectId).Execute()
```

Comment:
Dont you need to enable the LoadBalancer first?

Reply:
Not sure, but moved to start